crates/biome_js_analyze/tests/specs/nursery/noStaticElementInteractions/invalid.jsx

```diff
@@ -0,0 +1,8 @@
+<div onClick={() => {}} />;
```

Comment:
Could you add more test case from [ESLint](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/974275353598e9407c76bd4a50c331a953755cee/__tests__/src/rules/no-static-element-interactions-test.js#L41-L284)?